src/app/tests/suites/commands/delay/DelayCommands.cpp

```diff
@@ -63,3 +63,15 @@ CHIP_ERROR DelayCommands::RunInternal(const char * command)
     VerifyOrReturnError(system(command) == 0, CHIP_ERROR_INTERNAL);
     return ContinueOnChipMainThread(CHIP_NO_ERROR);
 }
+
+CHIP_ERROR DelayCommands::BusyWaitFor(uint32_t durationInMs)
+{
+    using namespace chip::System::Clock::Literals;
```

Comment:
This is no longer using the Literals thing for anything, since it's no longer using the `_ms` suffix.